docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/pypa/manylinux1_x86_64
```

Comment:
Agree, moved Dockerfile into dedicated manylinux subdirectory. That's the practice popular container authors follow. That's also reasonable because when creating a new image all the files from the Dockerfiles dir are packed with the image, so later we can add files into the subdirectory if we need them to have inside of container.